src/AccessPoint/AccessPoint.ts

```diff
@@ -57,7 +59,28 @@ export interface AccessPointFactory {
 /**
  * Constant that implements the {@link AccessPointFactory} interface.
  */
-export const AccessPoint:AccessPointFactory = {
+export const AccessPoint: {
+
+	/**
+	 * Specifies the type of the  object, in this case  'https://carbonldp.com/ns/v1/platform#AccessPoint'
+	 */
+	TYPE: C["AccessPoint"];
+
+	/**
+	 * Returns true when the value provided is considered to be a {@link AccessPoint}.
+	 */
+	is(value:any):value is AccessPoint;
+
+	/**
+	 * Creates a {@link AccessPoint} with the provided data.
+	 */
+	create<T extends object>( data:T & BaseAccessPoint ): T & TransientAccessPoint;
+
+	/**
+	 * Creates a {@link AccessPoint} from the provided Document.
+	 */
+	createFrom<T extends object>( object:T & BaseAccessPoint ):T & TransientAccessPoint;
+} = {
```

Comment:
Add a casting to the `AccessPointFactory`, to ensure the correct implementation in the constant.
```suggestion
} = <AccessPointFactory> {
```

(Follow this change for all the constants)